pkg/controllers/capr/rkecontrolplane/controlplane.go

```diff
@@ -103,7 +103,7 @@ func (h *handler) doRemove(cp *rkev1.RKEControlPlane) func() (string, error) {
 	return func() (string, error) {
 		logrus.Debugf("[rkecontrolplane] (%s/%s) Peforming removal of rkecontrolplane", cp.Namespace, cp.Name)
 		// Control plane nodes are managed by the control plane object. Therefore, the control plane object shouldn't be cleaned up before the control plane nodes are removed.
-		machines, err := h.machineCache.List(cp.Namespace, labels.SelectorFromSet(labels.Set{capi.ClusterLabelName: cp.Name, capr.ControlPlaneRoleLabel: "true"}))
+		machines, err := h.machineCache.List(cp.Namespace, labels.SelectorFromSet(labels.Set{capi.ClusterNameLabel: cp.Name, capr.ControlPlaneRoleLabel: "true"}))
```

Comment:
Does CAPI handle migrations of this label automatically? i.e. was upgrade testing performed?

Reply:
@jakefhyde , it's just the `const` name that changed in new version of CAPI it looks like.
Both `capi.ClusterLabelName` from pre-update and `capi.ClusterNameLabel` have the same value `cluster.x-k8s.io/cluster-name` so there is no "migration" to be done / upgrade testing to perform.

Am I missing something subtle here?

Reply:
This was just a constant name change in capi so I think it should be okay.

Reply:
Could've sworn I deleted this comment prior to leaving my review, oh well.